### ISSUE TITLE
[OS X][UI Request] Add a /Volumes/ shortcut.

### ISSUE THREAD

**Issue by Duamer** [Syncplay/syncplay#149]
Small request. Add a way to easily get to the /Volumes/ directory in the "Set media directories" interface. My workaround was to manually type the path, but many users wouldn't think to do that.

I would suggest adding a shortcut in the "Find Directory" window and an exception so that same window can see the directory in the root of the drive.

**Comment by alby128** [comment]
From a strictly technical point of view, this is possible, although it requires a fairly large amount of code rewriting. The self-initializing function getExistingDirectory needs to be replaced with a custom QFileDialog, encapsulated in a new method, and manually designed to behave in the same way, so to select only directories. Then, the /Volumes/ path could be added in it by using setSidebarUrls.

But, we should not forget that this issue was actually created by the use of non-native QFileDialogs instead of native dialogs, and that we had to introduce it as a workaround for some obscure qt4 bug on macOS. In macOS native file dialogs, this issue does not exist because the Finder shows all the mounted volumes in its sidebar.

Fixing this issue will require writing a substantial amount of macOS specific code that
- could stop working in the next major macOS release in a few days
- violates Apple's Human Interface Guidelines as it exposes the /Volumes/ folder to the GUI
- will become obsolete the day Syncplay migrates to qt5.

So, at this moment, I think this issue should not be fixed. I am of course open to suggestions.

**Comment by Et0h** [comment]
@alby128 @Duamer Now that we have Qt.py support is this fully resolved?

**Comment by alby128** [comment]
It will be resolved if and only if he uses PySide2 as binding.

**Comment by Et0h** [comment]
@alby128 Okay, so I guess that before closing this issue we need a short sentence explaining the problem and how it can be resolved for the Troubleshooting guide at http://syncplay.pl/guide/trouble/

**Comment by alby128** [comment]
@Et0h I tend to agree, although we are not providing anymore a guide to manually install Syncplay on macOS, so this possibility will be available only for experienced users. Maybe we should just leave this issue open until we switch to PySide2 in packaged .exe and .dmg .

**Comment by Et0h** [comment]
@alby128 Happy to keep this open until we have it packaged in .dmg. Not sure why there would be a need for it to be packaged in the Windows .exe, though.

**Comment by alby128** [comment]
@Et0h I just thought you wanted to keep all the platforms on the same binding, but that is not really necessary, you are right.
